lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +207,39 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        links = article_bs.find_all('a')
+        for link in links:
+            url = link['href']
+            if url.count('/') == 4 and url[0] == '/':
```

Comment:
are there any more specific checks? maybe there is something special between these slashes? otherwise the check is too vague